config/reports_kit/reports/total_tanks_by_facility.yml

```diff
@@ -0,0 +1,9 @@
+# what we're counting/aggregating (y-axis)
+measure: tank
+
+contextual_filters:
+- for_organization
```

Comment:
This, along with passing in the `context_params` would be required to provide the organization scoping desired.